asserts/asserts.go

```diff
@@ -278,6 +278,62 @@ func (ref *Ref) Resolve(find func(assertType *AssertionType, headers map[string]
 
 const RevisionNotKnown = -1
 
+// AtSequence represents a sequence forming assertion at a given sequence possibly
```

Comment:
I'm not sure why I used "represents" for AtRevision,  it would seem "references" would be a better verb.
also s/at a given sequence/at a given sequence point/